CONTRIBUTIONS.md

```diff
@@ -0,0 +1,63 @@
+### Guidelines for Pull Requests and Releases (Web3 1.x)
+
+This document provides some ground rules for contributors (including the maintainer(s) of
+the project) about how to make, review and publish changes to 1.x. The most basic requirement is
+that **Web3 not break**.
+
+## Pull Requests for substantive changes (e.g. everything except comments and docs)
+
+1. Any PR that introduces a logic change should include tests. (In many cases, the tests will take
+more time to write than the actual code).
+
+2. All PRs should sit for 48 hours with the `pleasereview` tag in order to garner feedback.
```

Comment:
I like that! It's way easier to see where reviews are required and where they aren't with this tag!